assets/app/components/nav.js

```diff
@@ -12,6 +12,8 @@ const propTypes = {
 const getSecondaryNav = (loggedIn, username) => {
   const authLinks = [
     <Link to="/sites">{username}</Link>,
+    <a href="https://federalist-docs.18f.gov" target="_blank">Documentation</a>,
```

Comment:
In the unauthenticated view, this link is titled "Help", but here it is titled "Documentation". Not asking for a change here, just wanted to make sure we realize and are okay with that.